sys/include/net/gcoap/fileserver.h

```diff
@@ -90,6 +90,50 @@ extern "C" {
  */
 #define COAPFILESERVER_DIR_DELETE_ETAG (0x6ce88b56u)
 
+/**
+ * @brief   GCoAP fileserver event types
+ *
+ * @note This requires the gcoap_fileserver_callback module.
+ */
+typedef enum {
+    GCOAP_FILESERVER_GET_START,     /**< file download started   */
+    GCOAP_FILESERVER_GET_END,       /**< file download finished  */
+    GCOAP_FILESERVER_PUT_START,     /**< file upload started     */
+    GCOAP_FILESERVER_PUT_END,       /**< file upload finished    */
+    GCOAP_FILESERVER_DELETE,        /**< file deletion requested */
```

Comment:
Please amend the doc: Is this called before or after the file was deleted?

Reply:
Ok, added